modules/network/src/main/java/org/apache/ignite/internal/network/netty/ConnectionManager.java

```diff
@@ -434,14 +436,10 @@ public void stop() {
 
         assert stopping.get();
 
-        Stream<CompletableFuture<Void>> stopFutures = Stream.concat(
-                Stream.concat(
-                    clients.values().stream().map(NettyClient::stop),
-                    Stream.of(server.stop())
-                ),
-                channels.values().stream().map(NettySender::closeAsync)
-        );
-        stopFutures = Stream.concat(stopFutures, Stream.of(disposeDescriptors()));
+        List<CompletableFuture<Void>> stopFutures = clients.values().stream().map(NettyClient::stop).collect(toList());
```

Comment:
Strictly speaking, it is incorrect to modify a list returned by `toList()`, it's better to create an `ArrayList` explicitly and use `forEach(list::add)`, for example